examples/taptree_of_horror/taptree_of_horror.rs

```diff
@@ -0,0 +1,314 @@
+        &[b0, b1, b2, b3, b4, b5, b6, b7, b8] => (b0, b1, b2, b3, b4, b5, b6, b7, b8),
+        _ => panic!("Expected exactly 8 elements in bob_pk_keys"),
+    };
+
+    let (c0, c1, c2, c3, c4, c5, c6, c7, c8) = match &c_pks[..] {
+        &[c0, c1, c2, c3, c4, c5, c6, c7, c8] => (c0, c1, c2, c3, c4, c5, c6, c7, c8),
+        _ => panic!("Expected exactly 8 elements in charlie_pk_keys"),
+    };
+
+    let (d0, d1, d2, d3, d4, d5, d6, d7, d8) = match &d_pks[..] {
+        &[d0, d1, d2, d3, d4, d5, d6, d7, d8] => (d0, d1, d2, d3, d4, d5, d6, d7, d8),
+        _ => panic!("Expected exactly 8 elements in dave_pk_keys"),
+    };
+
+    let (e0, e1, e2, e3, e4, e5, e6, e7, e8) = match &e_pks[..] {
+        &[e0, e1, e2, e3, e4, e5, e6, e7, e8] => (e0, e1, e2, e3, e4, e5, e6, e7, e8),
+        _ => panic!("Expected exactly 8 elements in eve_pk_keys"),
+    };
+
+    let (f0, f1, f2, f3, f4, f5, f6, f7, f8) = match &f_pks[..] {
+        &[f0, f1, f2, f3, f4, f5, f6, f7, f8] => (f0, f1, f2, f3, f4, f5, f6, f7, f8),
+        _ => panic!("Expected exactly 8 elements in frank_pk_keys"),
+    };
+
+    let (h0, h1, h2, h3, h4, h5, h6, h7, h8) = match &h_pks[..] {
+        &[h0, h1, h2, h3, h4, h5, h6, h7, h8] => (h0, h1, h2, h3, h4, h5, h6, h7, h8),
+        _ => panic!("Expected exactly 8 elements in heather_pk_keys"),
+    };
+
+    let (i0, i1, i2, i3, i4, i5, i6, i7, i8) = match &i_pks[..] {
+        &[i0, i1, i2, i3, i4, i5, i6, i7, i8] => (i0, i1, i2, i3, i4, i5, i6, i7, i8),
+        _ => panic!("Expected exactly 8 elements in ian_pk_keys"),
+    };
+
+    let (j0, j1, j2, j3, j4, j5, j6, j7, j8) = match &j_pks[..] {
+        &[j0, j1, j2, j3, j4, j5, j6, j7, j8] => (j0, j1, j2, j3, j4, j5, j6, j7, j8),
+        _ => panic!("Expected exactly 8 elements in judy_pk_keys"),
+    };
+
+    let (l0, l1, l2, l3, l4, l5, l6, l7, l8) = match &l_pks[..] {
+        &[l0, l1, l2, l3, l4, l5, l6, l7, l8] => (l0, l1, l2, l3, l4, l5, l6, l7, l8),
+        _ => panic!("Expected exactly 8 elements in liam_pk_keys"),
+    };
+
+    let (_s0, _s1, s2, _s3, s4, s5, _s6, s7, s8) = match &s_pks[..] {
+        &[_s0, _s1, s2, _s3, s4, s5, _s6, s7, s8] => (_s0, _s1, s2, _s3, s4, s5, _s6, s7, s8),
+        _ => panic!("Expected exactly 8 elements in secret1_pk_keys"),
+    };
+
+    let (_x0, _x1, x2, _x3, x4, x5, x6, _x7, x8) = match &x_pks[..] {
+        &[_x0, _x1, x2, _x3, x4, x5, x6, _x7, x8] => (_x0, _x1, x2, _x3, x4, x5, x6, _x7, x8),
+        _ => panic!("Expected exactly 8 elements in xecret2_pk_keys"),
+    };
+
+    // hashes that will also be used in the policy.
+    let g = grim.1;
+    let k = kelly.1;
+    // absolute timelocks that will be used in the policy
+    let oct_23_morning = "1729692000"; // Oct 23, 10:00 AM EST
+    let oct_24_evening = "1729819800"; // Oct 24, 09:30 PM EST
+    let oct_25_afternoon = "1729877400"; // Oct 25, 01:30 PM EST
+    let oct_26_morning = "1729942200"; // Oct 26, 07:30 AM EST
```

Comment:
In 1282d30653a3cb4e188e59d5a3dd8de909aa9477:

All these can just be numbers rather than strings. (It'll save you some .parse calls below).